scrapers/Boobpedia.yml

```diff
@@ -22,48 +22,72 @@ xPathScrapers:
+                with: $1
           - parseDate: January 2 2006
-      Ethnicity: //table//tr/td/b[text()='Ethnicity:']/../following-sibling::td/a
-      Country: //table//tr/td/b[text()='Nationality:']/../following-sibling::td/a
-      EyeColor: //table//tr/td/b[text()='Eye color:']/../following-sibling::td
+      Ethnicity: //table//tr/td/b[text()='Ethnicity']/../following-sibling::td/a
+      Country: //table//tr/td/b[text()='Nationality']/../following-sibling::td/a
+      EyeColor: //table//tr/td/b[text()='Eye color']/../following-sibling::td
       Height:
-        selector: //table//tr/td/b[text()='Height:']/../following-sibling::td
+        selector: //table//tr/td/b[text()='Height']/../following-sibling::td
         postProcess:
           - replace:
               - regex: (?:.+\D)?(\d+\.\d+)\Dm.+
                 with: $1
               - regex: \.
                 with: ""
       Weight:
-        selector: //table//tr/td/b[text()='Weight:']/../following-sibling::td
+        selector: //table//tr/td/b[text()='Weight']/../following-sibling::td
         postProcess:
           - replace:
               - regex: (?:.+\D)?(\d+)\Dkg.+
                 with: $1
       Measurements:
-        selector: //table//tr/td/b[text()='Measurements:']/../following-sibling::td|//table//tr/td[contains(b,'cup')]/following-sibling::td
+        selector: //table//tr/td/b[text()='Measurements']/../following-sibling::td|//table//tr/td[contains(b,'cup')]/following-sibling::td
         concat: "|"
         postProcess:
           - replace:
-              - regex: (\d+)-(\d+)-(\d+)[^|]+\|(\S+).+ # get measurements + cup
-                with: $1$4-$2-$3
+              - regex: (\d+)-(\d+)-(\d+)[^|]*\|(\S+).+ # get measurements + cup
+                with: $4-$2-$3
               - regex: \|.+$ # fallback to clear non matching regexes
                 with: ""
-      FakeTits: //table//tr/td/b[text()='Boobs:']/../following-sibling::td/a
-      HairColor: //table//tr/td[contains(b,'Hair')]/following-sibling::td
+              - regex: \[\d*\] # Remove References
+                with: ""
+              - regex: ( in) # Remove Unit Inches
+                with: ""
+      FakeTits: //table//tr/td/b[text()='Boobs']/../following-sibling::td/a
+      HairColor:
+        selector: //table//tr/td[contains(b,'Hair')]/following-sibling::td//text()
+        concat: ", "
+        postProcess:
+          - replace:
+              - regex: (,,)
+                with: ","
+              - regex: ( , )
+                with: " "
       # nbsp; screws up the parsing, so use contains instead
-      CareerLength: //table//tr/td/b[text()[contains(.,'active:')]]/../following-sibling::td
+      CareerLength:
+        selector: //table//tr/td/b[text()[contains(.,'active')]]/../following-sibling::td
+        postProcess:
+          - replace:
+              - regex: (present)
+                with: "Present"
+              - regex: (current)
+                with: "Present"
```

Comment:
I personally prefer to just leave it empty after `-` instead of using the word `Present`>.

It might also mess with the draft submissions <https://github.com/stashapp/stash/pull/3146>.

Reply:
I personally prefer "Present", looks a bit nicer imho :-P

However, I see your point with the linked PR, so I changed the regex accordingly.